conans/test/functional/toolchains/cmake/test_cmake.py

```diff
@@ -528,3 +528,53 @@ def build(self):
         client.run("install .")
         client.run("build .")
         self.assertIn("VALUE OF CONFIG STRING: my new value", client.out)
+
+
+@pytest.mark.skipif(platform.system() != "Windows", reason="Tests Windows MinGW")
```

Comment:
``TestSubsystemsCMakeBuild`` is already building very similar tests, isn't there any overlap?

Reply:
Still a draft. I made another test for better isolation of the test but I will unify them to the existing MinGW ones once I figure out the cmake behavior